stats.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional, Dict, Union
+
+StatsAnnot = Dict[str, Union[int, float]]
+
+class Stats:
+    """
+    Champions, items, and rune shards can all be considered to have stats.
+    Stats in league all have a fixed way of being stacked, hence why it makes sense to define a class.
+    """
+
+    def __init__(self, stat_dict: Optional[StatsAnnot] = None):
+        if stat_dict is None:
+            stat_dict = {}
+            
+        for name, value  in stat_dict.items():
+            setattr(self, name, value)
+
+    def print_stats(self):
+        return print("\n".join([f"{k}: {v}" for k, v in self.__dict__.items()]))
+
+
+    def __add__(self, stats: StatsAnnot):
+        addition = dict()
+        for name, value in stats.__dict__.items():
```

Comment:
`stats` is already a dict no? `stats.items()`

Reply:
stats would be a Stats object in this case (to do stat1 + stat2). Attribute wise addition / or setting if the attribute does not exist in stat1.

Reply:
Ok, in this case the type annotation is wrong

Reply:
Indeed, I mixed things up